examples/src/main/java/ai/djl/examples/training/TrainBertOnCode.java

```diff
@@ -73,10 +73,12 @@ public static TrainingResult runExample(String[] args) throws IOException, Trans
         }
     }
 
-    private static Model createBertPretrainingModel(int dictionarySize) {
+    private static Model createBertPretrainingModel(long vocabularySize) {
         Block block =
                 new BertPretrainingBlock(
-                        BertBlock.builder().micro().setTokenDictionarySize(dictionarySize));
+                        BertBlock.builder()
+                                .micro()
+                                .setTokenDictionarySize(Math.toIntExact(vocabularySize)));
```

Comment:
Should we also change the .setTokenDictionarySize() function name?

Reply:
I'll do this in a later PR